scripts/local_testnet/network_params.yaml

```diff
@@ -1,12 +1,32 @@
 # Full configuration reference [here](https://github.com/ethpandaops/ethereum-package?tab=readme-ov-file#configuration).
 participants:
+  # Stateful proof generator nodes (generate proofs for the network)
   - el_type: geth
     el_image: ethereum/client-go:latest
     cl_type: lighthouse
     cl_image: lighthouse:local
     cl_extra_params:
       - --target-peers=3
-    count: 4
+      - --generate-execution-proofs
+    count: 1
+  # Stateless validator nodes (consume proofs but cannot generate them)
+  - el_type: geth
+    el_image: ethereum/client-go:latest
+    cl_type: lighthouse
+    cl_image: lighthouse:local
+    cl_extra_params:
+      - --target-peers=3
+      - --stateless-validation
+      - --stateless-min-proofs-required=3
+    count: 2
+  # Regular stateful nodes (normal operation, no proof generation)
+  - el_type: geth
+    el_image: ethereum/client-go:latest
+    cl_type: lighthouse
+    cl_image: lighthouse:local
+    cl_extra_params:
+      - --target-peers=3
+    count: 1
```

Comment:
You should add a new kurtosis params file. If you want to test this in CI add a new job